packages/proxy/schema/secrets.ts

```diff
@@ -0,0 +1,74 @@
+import { z } from "zod";
+
+export const BaseMetadataSchema = z
```

Comment:
This wasn't strictly necessary, but the validation logic for model metadata is getting quite interesting, and I figured it was a good time to upgrade it to Zod.